README.md

```diff
@@ -18,7 +18,7 @@ _Collaborate and work together on GitHub._
   Do not use quotes on the <details> tag attributes.
 -->
 
-<details id=0 open>
+<details id=0>
 <summary><h2>Welcome</h2></summary>
 
```

Comment:
Pull requests facilitate collaboration by allowing developers to share their work with others and receive feedback and suggestions. This can lead to better code quality, as issues can be caught and addressed early on.